arm/.global/global.module.tests.ps1

```diff
@@ -269,29 +269,46 @@ Describe 'Readme tests' -Tag Readme {
             $parameters = $templateContent.parameters.Keys
 
             # Get ReadMe data
+            ## Get section start index
             $parametersSectionStartIndex = 0
             while ($readMeContent[$parametersSectionStartIndex] -notlike '*# Parameters' -and -not ($parametersSectionStartIndex -ge $readMeContent.count)) {
                 $parametersSectionStartIndex++
             }
-
-            $parametersTableStartIndex = $parametersSectionStartIndex + 1
-            while ($readMeContent[$parametersTableStartIndex] -notlike '*|*' -and -not ($parametersTableStartIndex -ge $readMeContent.count)) {
-                $parametersTableStartIndex++
-            }
+            Write-Verbose ("parametersSectionStartIndex $parametersSectionStartIndex") -Verbose
```

Comment:
Leftover?

Reply:
nope

Reply:
I could maybe comment that out but would keep it "at hand" until all readmes are updated, in case troubleshooting is needed

Reply:
I vote to either remove it, or remove the `-Verbose`

```suggestion
            Write-Verbose ("parametersSectionStartIndex $parametersSectionStartIndex")
```